test/e2e/specs/editor/various/pattern-overrides.spec.js

```diff
@@ -269,9 +269,16 @@ test.describe( 'Pattern Overrides', () => {
 
 			await editor.setContent( '' );
 
+			// Insert the core/group block with a tagName of 'main'
 			await editor.insertBlock( {
-				name: 'core/block',
-				attributes: { ref: id },
+				name: 'core/group',
+				attributes: { tagName: 'main' },
+				innerBlocks: [
+					{
+						name: 'core/block',
+						attributes: { ref: id },
```

Comment:
I'm not sure about this. I suspect the failures to be something to do with playwright selectors, since if I stop the test here and interact with the page the `inert` attributes behave as expected. As the do with an identical set up in any site.

Instead of wrapping the pattern block, I just appended a group main block to the page and went from there:

```js
			await editor.insertBlock( {
				name: 'core/group',
				attributes: { tagName: 'main' },
				innerBlocks: [
					{
						name: 'core/paragraph',
						content: 'Main group block so the zoom toggle appears.',
					},
				],
			} );

```

Furthermore, the reason why I suspect playwright is that the first `Zoomed in / Design mode` block passes when  I add `await editor.canvas.locator( 'body' ).click();`. 

 

Maybe the test needs to be set up differently 🤷🏻🤷🏻🤷🏻  @talldan will some pointers perhaps.

Reply:
Took me a while to understand what's going on. I think this change seems ok, a section root is required so this line is adding it. The comment could perhaps be clearer to explain that a section root is being added.

The reason the test fails now is that the group block is the selected block after this change, while in `trunk` the pattern block is the selected block. The pattern block has the 'clickthrough' behavior when unselected, and that sets `inert` on all the inner blocks of the pattern.

I'll push a commit to fix. I think it might be good to take advantage this and add some assertions about the click through behavior to the test.

Reply:
That's great sleuthing there Dan. Thank you for unpicking this 👏

Reply:
> I think it might be good to take advantage this and add some assertions about the click through behavior to the test.

It looks like you did this already. Hopefully I wasn't wrong there.

Reply:
Yep 👍